src/mode/graphics.rs

```diff
@@ -37,6 +37,7 @@ where
     }
 
     /// Reset display
+    // TODO: Move to a more appropriate place
```

Comment:
Hmm, yes this is a strange one. It's only applicable to the SPI interface which makes placing it in the architecture of this crate pretty weird. Do you have any thoughts where this could go?